### PR TITLE
Remove deprecated no-unused-variable rule

### LINKED ISSUE #49: Remove no-unused-variable (deprecated in tslint)
Current versions of tslint emit the following deprecation warning:

    no-unused-variable is deprecated. Since TypeScript 2.9. Please use the built-in compiler checks instead.

### PR DESCRIPTION
**Pull request by jrobeson**
`no-unused-variable` was deprecated in tslint 5.11.0

Fixes #49

### CODE REVIEW

**Comment by jrobeson** [comment]
oops, guess that's not right, let me try again.

**Comment by jrobeson** [comment]
Oh i see that this was already failing. https://travis-ci.org/blakeembrey/tslint-config-standard/builds

**Comment by blakeembrey** [comment]
Thanks @jrobeson, I'll need to look into the failures again when I have time 😄
